### ISSUE TITLE
different Learning rate images

### ISSUE THREAD

**Issue by Muthukamalan** [Muthukamalan/ERAV1#3]
![backprop eta=1](https://github.com/Muthukamalan/ERAV1/assets/50898904/87ac0263-6e7a-4da3-8dff-e4a89d29da9a)

![backprop eta=10](https://github.com/Muthukamalan/ERAV1/assets/50898904/81cd1486-1420-4c32-ad35-b6250f3a9e44)

**Comment by Muthukamalan** [comment]
able to add in readme closing issues
